### PR TITLE
updates code to aws s3/ses flow

### LINKED ISSUE #8: Send email after placing an order

### PR DESCRIPTION
**Pull request by malczak**
- some changes for email sending flow 
Email sending is done on AWS with S3/SES events 

This PR closes #8 

💣💣💣 will not work on now - we need to update AWS config 💣💣💣

### CODE REVIEW

**Comment by sobstel** [comment]
> 💣💣💣 will not work on now - we need to update AWS config 💣💣💣

send me config vars please and we'll test it at ~now~ vercel (ugh) preview.

File: components/basket/Summary.tsx [hunk 1]
```diff
@@ -9,24 +9,25 @@ const ClientInfo: React.FC<{ client: Client }> = ({ client }) => (
...
+  client?: Client;
+  onOrder: () => void;
+}> = ({ items, client, onOrder }) => {
   return (
     <div>
       <ClientInfo client={client} />
       <Cart items={items} />
       <div className="tw-my-4 tw-mx-6 tw-flex tw-justify-center">
-        <Button type="primary" shape="round">
+        <Button type="primary" shape="round" onClick={() => onOrder()}>
```
Comment (sobstel) [inline components/basket/Summary.tsx:30]: `onClick={onOrder}` won't work?

File: components/order/Printer.tsx [hunk 2]
```diff
@@ -38,15 +38,13 @@ export class OrderPrinter extends React.PureComponent<{ order: Order }> {
                 <table>
                   <tr>
                     <td>
-                      Fabristic
+                      {brand.name}
                       <br />
-                      The Pirate Cat
+                      Przykładowa, 9/11
```
Comment (sobstel) [inline components/order/Printer.tsx]: I think we can skip the address and NIP number. I don't want it to look like a real invoice.

**Review by sobstel — CHANGES_REQUESTED** [review]
Needs a preview test first (blocked by missing AMZ env vars?).

File: components/basket/Summary.tsx (hunk 1 above)
Comment (malczak) [reply components/basket/Summary.tsx:30]: Would work but TS is complaining about proper event types 🤮

File: components/order/Printer.tsx (hunk 2 above)
Comment (malczak) [reply components/order/Printer.tsx]: As you wish it should go to brand config anyways

**Pull request merged**